_episodes/01-conda.md

```diff
@@ -58,27 +59,28 @@ where the community can contribute conda installation packages.
 You can search Anaconda Cloud
 to find the command line entry needed to install the package. e.g:
 ~~~
-$ conda install -c https://conda.anaconda.org/scitools iris
+$ conda install -c conda-forge iris
 ~~~
 {: .language-bash}
 
-In many cases, there are many versions of the same package up on Anaconda Cloud.
-To address this problem, [conda-forge](https://conda-forge.github.io/)
-has been launched to have a central place for just a single version.
-You can therefore expand the selection of packages available via `conda install`
-beyond the chosen 330 by adding conda-forge:
+In many cases there are many versions of the same package up on Anaconda Cloud.
+We recommend to add the  [conda-forge](https://conda-forge.github.io/),
+in additional to the default channel, to avoid mixing packages from multiple third party channels.
```

Comment:
I don't want to get into the details of the pitfalls of using multiple channels, but we should not encourage users to do that, specially beginners. For more info we can maybe point them to https://conda-forge.org/docs/conda-forge_gotchas.html#using-multiple-channels

Reply:
I like this recommendation.